tests/50federation/35room-invite.pl

```diff
@@ -0,0 +1,134 @@
+test "Outbound federation can send invites",
+   requires => [ local_user_and_room_fixtures(), $main::INBOUND_SERVER, federation_user_id_fixture() ],
+
+   do => sub {
+      my ( $user, $room_id, $inbound_server, $invitee_id ) = @_;
+
+      Future->needs_all(
+         $inbound_server->await_request_invite( $room_id )->then( sub {
+            my ( $req, undef ) = @_;
+
+            assert_eq( $req->method, "PUT",
+               'request method' );
+
+            my $body = $req->body_from_json;
+            log_if_fail "Invitation", $req->body_from_json;
+
+            # this should be a member event
+            assert_json_keys( $body, qw( event_id origin room_id sender type ));
+
+            assert_eq( $body->{type}, "m.room.member",
+               'event type' );
+            assert_eq( $body->{origin}, $user->http->server_name,
+               'event origin' );
+            assert_eq( $body->{room_id}, $room_id,
+               'event room_id' );
+            assert_eq( $body->{sender}, $user->user_id,
+               'event sender' );
+
+            assert_json_keys( $body, qw( content state_key prev_state ));
+
+            assert_deeply_eq( $body->{content}, { membership => "invite" },
+               'event content' );
+            assert_eq( $body->{state_key}, $invitee_id,
+               'event state_key' );
+
+            $inbound_server->datastore->sign_event( $body );
+
+            $req->respond_json(
+               # SYN-490
+               [ 200 => { event => $body } ]
```

Comment:
Eh; it doesn't actually matter as they do the same thing here. But yes for consistency I guess it should be `,`.  Done c3f2eb1